### PR TITLE
Adds GitLab to the Continuous Integration page

### PR DESCRIPTION
**Pull request by samdbeckham**
Does pretty much what it says in the title.

screenshot 2019-02-20 at 17 27 21

### CODE REVIEW

File: lang/en/docs/_ci/gitlab.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - yarn install
+
+test:
+  stage: test
+  cache:
+    paths:
+    - node_modules/
+    - .yarn
+  script:
+    - yarn test-ci
```
Comment (Daniel15) [inline lang/en/docs/_ci/gitlab.md]: Is `test-ci` specific to your project? Not sure this is a general thing we should mention in the docs.

Comment (samdbeckham) [reply lang/en/docs/_ci/gitlab.md]: It's just an example of something you could run. Though you're right it might lead to confusion. I'll pull it out  🙂

**Comment by Haroenv** [comment]
thanks!

**Comment by samdbeckham** [comment]
Thanks for merging @Haroenv 😄 

It doesn't seem to have worked on the build for some reason. The content isn't showing up

https://yarnpkg.com/en/docs/install-ci

**Comment by Haroenv** [comment]
I probably implemented the conditional in https://github.com/yarnpkg/website/commit/fe50d77f15a90dbb27be13b7edb6ee59ef1a402b wrong

**Comment by samdbeckham** [comment]
That looks a bit strange. Why is the GitLab implementation handled differently to the others? 🤔

**Comment by Haroenv** [comment]
Because the translations didn’t have the partial yet, so it would fail to build. I’ll look into it!

**Pull request merged**